janusgraph-driver/src/main/java/org/janusgraph/core/attribute/Text.java

```diff
@@ -254,16 +258,15 @@ public String toString() {
      * @return true if {@code term} is similar to {@code value} 
      */
     private static boolean isFuzzy(String term, String value){
-        int distance;
         term = term.trim();
         if (term.length() < 3) {
-            distance = 0;
+            return ZERO_LEVENSHTEIN_DISTANCE.apply(value, term) == 0;
```

Comment:
Just thinking. Won't it be a little bit more efficient if we would use `return value.equals(term);` here?
As I understand `LevenshteinDistance` is 0 only for equal strings, if I am not missing anything.

Reply:
Ouch, why didn't I think about this in the first place... I think you are right. I'll update it later today.